Sources/PubNub/Helpers/PAMTokenStore.swift

```diff
@@ -165,14 +165,14 @@ public struct PAMTokenResource: Codable, Equatable, Hashable {
   public let channels: [String: PAMPermission]
   public let groups: [String: PAMPermission]
 
-  public let users: [String: PAMPermission]
-  public let spaces: [String: PAMPermission]
+  public let uuidObjects: [String: PAMPermission]
+  public let channelObjects: [String: PAMPermission]
 
   enum CodingKeys: String, CodingKey {
     case channels = "chan"
     case groups = "grp"
-    case users = "usr"
-    case spaces = "spc"
+    case uuidObjects = "usr"
```

Comment:
Should this be `case uuidObjects = "uuid"?

Reply:
I couldn't find a PRD for any PAM v3 changes, so I based it on the [JS SDK](https://github.com/pubnub/javascript/blob/master/src/core/components/token_manager.js#L181-L184) not updating the key fields.